README.md

```diff
@@ -279,6 +279,21 @@ let json = to_json B
 (* "bbb" *)
 ```
 
+#### `[@@deriving json_string]`: a shortcut for JSON string conversion
+
+For convenience, one can use `[@@deriving json_string]` to generate converters
+directly to and from JSON strings:
+
+```ocaml
+type t = A [@@deriving json, json_string]
```

Comment:
Is it necessary to add `json` whenever `json_string` is used? Otherwise, does it make sense to remove it from this example?

Reply:
Yes, it's necessary, the `json_string` deriver depends on `json`. If one omits it then they will see an error message stating such.